pyinteraph/path_analysis.py

```diff
@@ -482,6 +482,32 @@ def reorder_graph(graph, identifiers):
     ordered_graph.add_weighted_edges_from(weighted_edges)
     return(ordered_graph)
 
+def write_metapath_table(graph, identifiers, residue_names, fname):
+        '''The function returns a file csv containing the information
+        about the edges of the metapath graph and their weight.'''
+
+        # Create dictionary of {ids : resnames}
+        nodes_dict = dict(zip(identifiers, residue_names))
+        
+        with open(fname, 'w') as out:
+            
+            # Write header
+            out.write('chain1,resid1,restype1,group1,' \
+                      'chain2,resid2,restype2,group2,weight \n')
```

Comment:
the name of the `weight` column should be more descriptive (i.e. `recurrency` or `relative_recurrency` depending on whether we normalise or not)